doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -82,77 +82,64 @@ Market configuration (beyond base type and quote type) consists of 3 integer val
-```
+Towards those ends, let's use 0.001 `eAPT` for the order size granularity, that is lot size precision.
+Let's also use 0.001 `eUSDC` for the price granularity, equivalent to one-hundredth of a penny.
+Last, we'll use a minimum order size of 0.5 `eAPT` so that small orders are allowed while preventing attacks.
 
-That's a price granularity of 1 cent, since the result here is in quote units and 1 `eUSDC` is $1.
-For good measure, we also checked the maximum price per `eAPT` using `get_max_quote_per_base_nominal`; it's $42,949,672.95 per `eAPT` so this is fine!
-Be careful: using a more granular price results in a lower maximum price, which may be relevant in some cases.
-Now that we have the minimum base and quote sizes we'd like to use, we're ready to configure the market.
-Let's get the lot size and tick size:
+The `get_market_parameters_integer` helper from the SDK will give us the configuration variables needed to achieve these outcomes:
 
-```
->>> lot_size = get_lot_size_integer("0.001", base_decimals)
->>> lot_size
-100000
->>> tick_size = get_tick_size_integer("0.00001", quote_decimals)
->>> tick_size
-10
+```python
+>>> base_decimals = 8
+>>> quote_decimals = 6
+>>> size_precision_nominal = "0.001"
+>>> price_precision_nominal = "0.001"
+>>> min_size_nominal = "0.5"
+>>> (lot_size, tick_size, min_size) = get_market_parameters_integer(
+... size_precision_nominal,
+... price_precision_nominal,
+... min_size_nominal,
+... base_decimals,
+... quote_decimals
+... )
+>>> (lot_size, tick_size, min_size)
+(100000, 1, 500)
 ```
 
-We also need a minimum (limit order) size.
-Let's say orders can only be submitted if they are for at least 1 whole `eAPT`, using the lot size of 100000 from above:
+Let's also check the maximum price given our price granularity:
 
-```
->>> get_min_size_integer("1.0", base_decimals, lot_size)
-1000
+```python
+>>> get_max_price_nominal(price_precision_nominal)
+Decimal('4294967.295')
 ```
 
-That means we use use a minimum size of 1000 lots to configure our market, so our market parameters would be:
+That's a maximum price of \$4,294,967.295 per `eAPT` which is plenty high.
+So, our market parameters would be:
 
 | Parameter | Value  | Units             |
 | --------- | ------ | ----------------- |
 | Lot size  | 100000 | Subunits of base  |
-| Tick size | 10     | Subunits of quote |
-| Min size  | 1000   | Lots of base      |
+| Tick size | 1      | Subunits of quote |
+| Min size  | 500    | Lots of base      |
 
-This gives us a market with a price granularity of 1 cent and minimum order size of 1 `eAPT`!
-Can you see how one would get a price granularity of 0.1 cents instead?
+This gives us a market with a price granularity of 0.1 cent and minimum order size of 0.5 `eAPT`!
+We'd get an error if we tried to use a more granular price, or more granular size without changing price granularity.
+This is because our tick size is 1, which represents the minimum--the highest possible granularity of the quote asset.
```

Comment:
How about?

"This is because our tick size is 1, which represents the most possible price precision for the given size and the highest possible granularity of the quote asset"

Reply:
done